ovh/resource_cloud_project_region_storage_presign_test.go

```diff
@@ -0,0 +1,74 @@
+package ovh
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"os"
+	"testing"
+)
+
+const testCloudProjectRegionStoragePresign = `
+resource "ovh_cloud_project_region_storage_presign" "presign_url" {
+  service_name = "%s"
+  region_name  = "%s"
+  name         = "%s"
+  expire       = 3600
+  method       = "GET"
+  object       = "%s"
+}
+`
+
+func testAccPreCheckCloudRegionStorage(t *testing.T) {
+	testAccPreCheckCredentials(t)
+	checkEnvOrSkip(t, "OVH_CLOUD_PROJECT_REGION_STORAGE_NAME_TEST")
+	checkEnvOrSkip(t, "OVH_CLOUD_PROJECT_REGION_STORAGE_BUCKET_NAME_TEST")
+	checkEnvOrSkip(t, "OVH_CLOUD_PROJECT_REGION_STORAGE_OBJECT_TEST")
+}
+
+func testAccCheckCloudRegionStorage(t *testing.T) {
+
+	type cloudProjectRegionStorageResponse struct {
+		Name   string `json:"name"`
+		Region string `json:"region"`
+	}
+
+	r := cloudProjectRegionStorageResponse{}
+
+	endpoint := fmt.Sprintf("/cloud/project/%s/region/%s/storage/%s",
+		os.Getenv("OVH_CLOUD_PROJECT_SERVICE_TEST"),
+		os.Getenv("OVH_CLOUD_PROJECT_REGION_STORAGE_NAME_TEST"),
+		os.Getenv("OVH_CLOUD_PROJECT_REGION_STORAGE_BUCKET_NAME_TEST"))
+
+	err := testAccOVHClient.Get(endpoint, &r)
+	if err != nil {
+		t.Fatalf("Error: %q\n", err)
+	}
+	t.Logf("Read Storage Container %s -> name: '%s', region: '%s'", endpoint, r.Name, r.Region)
+}
+
+func TestCloudProjectRegionStoragePresign(t *testing.T) {
+	serviceName := os.Getenv("OVH_CLOUD_PROJECT_SERVICE_TEST")
+	regionName := os.Getenv("OVH_CLOUD_PROJECT_REGION_STORAGE_NAME_TEST")
```

Comment:
regarding environment variables, for databases we already have:
`OVH_CLOUD_PROJECT_DATABASE_REGION_TEST`

For kubernetes:
`OVH_CLOUD_PROJECT_KUBE_REGION_TEST`

So it should be more convenient to follow te same convention for:
`OVH_CLOUD_PROJECT_REGION_STORAGE_NAME_TEST`

Rename the env variable to: `OVH_CLOUD_PROJECT_STORAGE_REGION_TEST`